website/docs/tools/analyzer.md

```diff
@@ -21,7 +21,6 @@ Currently, the following package managers (grouped by the programming language t
 * Go
   * [dep](https://golang.github.io/dep/)
   * [Glide](https://github.com/Masterminds/glide)
-  * [Godep](https://github.com/tools/godep)
```

Comment:
Glide and dep should be removed as well, as these were also handled by the Godep implementation.